great_expectations_cloud/agent/agent.py

```diff
@@ -176,9 +176,7 @@ def _handle_event_as_thread_enter(self, event_context: EventContext) -> None:
             # this event has been redelivered too many times - remove it from circulation
             event_context.processed_with_failures()
             return
-        elif self._can_accept_new_task() is not True or isinstance(
-            event_context.event, UnknownEvent
-        ):
+        elif self._can_accept_new_task() is not True:
             # request that this message is redelivered later. If the event is UnknownEvent
             # we don't understand it, so requeue it in the hope that someone else does.
```

Comment:
Minor nit: Does this comment still apply? If not let's remove it or update it.

Reply:
updated in 477073b